onnx/src/ops/array/mod.rs

```diff
@@ -35,6 +36,7 @@ pub fn register_all_ops(reg: &mut OnnxOpRegister) {
     reg.insert("Squeeze", squeeze);
     reg.insert("Tile", |_, _| Ok((expand(array::Tile::default()), vec![])));
     reg.insert("Transpose", transpose);
+    reg.insert("Range", range::range);
```

Comment:
Try to see in harness/onnx-test-suite if the `include-passing-ignored.sh` picks up more passing tests with Range and adds them to the list of passing tests.

You can also run `cargo test --ignored range`.